spec/lib/serializable_spec.rb

```diff
@@ -51,6 +51,11 @@ class UserSerializer
       expect(result).to eq(user_data)
     end
 
+    it "serialize nil, under [:data] if resource is nil" do
```

Comment:
Can you add a test case for `serializable_class` as well, even though the behavior hasn't changed?